src/core/org/apache/jmeter/resources/messages_fr.properties

```diff
@@ -853,7 +853,7 @@ reportgenerator_summary_statistics_kbytes=Ko/sec
 reportgenerator_summary_statistics_label=Libell\u00E9
 reportgenerator_summary_statistics_max=Max
 reportgenerator_summary_statistics_min=Min
-reportgenerator_summary_statistics_percentile_fmt=%d%% Ligne
+reportgenerator_summary_statistics_percentile_fmt=%de percentile
```

Comment:
Hi Milamber,

I am ok with your but in wikipedia (https://fr.wikipedia.org/wiki/Centile)
I have seen that percentile is also accepted

I quoted wikipedia:
"En statistique descriptive
<https://fr.wikipedia.org/wiki/Statistique_descriptive>, un *centile* est
chacune des 99 valeurs qui divisent les données triées en 100 parts égales,
de sorte que chaque partie représente 1/100 de l'échantillon de population.
(La dénomination anglaise <https://fr.wikipedia.org/wiki/Anglais> —
*percentile*, avec une prononciation francisée, est parfois utilisée)."

It's why I have prefered percentile instead of centile

But if you prefer centile, it's ok for me too

Antonio


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=oa-2200-c>
Garanti
sans virus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=oa-2200-c>
<#DDB4FAA8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

2016-04-09 9:23 GMT+02:00 Milamber <notifications@github.com>:

> In src/core/org/apache/jmeter/resources/messages_fr.properties
> <https://github.com/apache/jmeter/pull/187#discussion_r59111727>:
>
> > @@ -853,7 +853,7 @@ reportgenerator_summary_statistics_kbytes=Ko/sec
> >  reportgenerator_summary_statistics_label=Libell\u00E9
> >  reportgenerator_summary_statistics_max=Max
> >  reportgenerator_summary_statistics_min=Min
> > -reportgenerator_summary_statistics_percentile_fmt=%d%% Ligne
> > +reportgenerator_summary_statistics_percentile_fmt=%de percentile
>
> In French, percentile is translate by the word "centile"
> https://fr.wikipedia.org/wiki/Centile
>
> For example : "99e centile"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/jmeter/pull/187/files/0165352fa09a2771b19290c72f5425a875cf1513#r59111727>
>